electron/config.ts

```diff
@@ -186,12 +188,41 @@ abstract class GlobalConfig {
       }
     })
 
+    const crossover: Set<WineInstallation> = new Set()
+
+    if (isMac)
+    {
+      execAsync('mdfind kMDItemCFBundleIdentifier = "com.codeweavers.CrossOver"')
+        .then(async ({ stdout }) => {
+          stdout.split('\n').forEach((crossoverMacPath) => {
+            if (crossoverMacPath != '' && existsSync(crossoverMacPath + '/Contents/Info.plist')) {
+              const info = plistParse(readFileSync(crossoverMacPath + '/Contents/Info.plist', 'utf-8')) as PlistObject
+              const version = info['CFBundleShortVersionString'] || ''
+              crossover.add({
+                bin: `'${crossoverMacPath}/Contents/SharedSupport/CrossOver/bin/wine'`,
+                name: `CrossOver ${version} - ${crossoverMacPath}`
+              })
+            }
+          })
+        })
+    }
+    else if (!isWindows)
```

Comment:
This would be for Crossover for Linux, right?

Reply:
Yes, I'll add a comment